Tests/Tests/Core/Utilities/Exif/ExifTests.swift

```diff
@@ -6,8 +6,10 @@
 //  Copyright © 2023 Povio Inc. All rights reserved.
 //
 
+#if os(iOS)
 import XCTest
 import PovioKitCore
+@testable import PovioKitUtilities
```

Comment:
Why using `@testable`? All APIs are public 🤔

Reply:
Removed where not necessary.